src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -120,110 +128,63 @@ const formConfig = {
   subTitle: 'Form 21-526EZ',
   // getHelp: GetFormHelp, // TODO: May need updated form help content
   chapters: {
-    reviewVeteranDetails: {
-      title: (isReviewPage) => {
-        const baseString = 'Review Veteran Details';
-        if (isReviewPage) {
-          return baseString.replace('Review ', '');
-        }
-        return baseString;
-      },
+    veteranDetails: {
+      title: (isReviewPage) => `${isReviewPage ? 'Review ' : ''}Veteran Details`,
       pages: {
-        veteranInformation: createVerifiedVeteranInfoPage(fullSchema526EZ),
-        specialCircumstances: { // TODO: create page file and reuse 
```

Comment:
This is a particularly ugly diff...Basically, there were "duplicate" pages in these two chapters, where one chapter had all the pages for verified and one had all the pages for unverified. I've only replaced the `veteranInformation` page for now, but have removed the other duplicate pages.